docs/attributes-registry/otel.md

```diff
@@ -1,46 +1,42 @@
 <!--- Hugo front matter used to generate the website version of this page:
 --->
 
-# OpenTelemetry
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
 
-<!-- toc -->
+# Otel
```

Comment:
Oh no :(

Reply:
This is fixable now - I'll look into using the new acronym registry feature

Reply:
Updated to use new acronym feature (cleans up AWS, EKS, OpenTracing etc. as well)